lib/productions/operation.js

```diff
@@ -45,6 +46,10 @@ export class Operation extends Base {
   }
 
   *validate(defs) {
+    if (!this.name && ["", "static"].includes(this.special)) {
+      const message = `Regular or static operations need a proper identifier.`;
```

Comment:
I think we should avoid "proper" as it's ambiguous. Maybe just: 

```suggestion
      const message = `Regular or static operations must have an identifier.`;
```

Reply:
Maybe it should say "Did you forget the return type?"

Reply:
That might be helpful.